From f.b.brokken@rc.rug.nl  Wed Aug 30 15:42:32 2006
Return-Path: <f.b.brokken@rc.rug.nl>
Received: from knarfix.oostum.north (localhost [127.0.0.1])
	by knarfix.oostum.north (8.13.7/8.13.7/Debian-2) with ESMTP id k7UDgWO0011811
	for <frank@localhost>; Wed, 30 Aug 2006 15:42:32 +0200
Received: from styx.oostum.north [192.168.17.2]
	by knarfix.oostum.north with POP3 (fetchmail-6.3.4)
	for <frank@localhost> (single-drop); Wed, 30 Aug 2006 15:42:32 +0200 (CEST)
Received: from oosix.icce.rug.nl (oosix.icce.rug.nl [129.125.14.80])
	by styx.oostum.north (8.13.7/8.13.7/Debian-2) with ESMTP id k7UDgQHL017806
	(version=TLSv1/SSLv3 cipher=DHE-RSA-AES256-SHA bits=256 verify=NOT)
	for <frank@styx.oostum.north>; Wed, 30 Aug 2006 15:42:26 +0200
Received: from suffix.rc.rug.nl (suffix.rc.rug.nl [129.125.3.162])
	by oosix.icce.rug.nl (8.13.7/8.13.7/Debian-1) with ESMTP id k7UDgL61032002
	(version=TLSv1/SSLv3 cipher=DHE-RSA-AES256-SHA bits=256 verify=NOT)
	for <frank2@oostum.nl>; Wed, 30 Aug 2006 15:42:21 +0200
Received: from suffix.rc.rug.nl (localhost [127.0.0.1])
	by suffix.rc.rug.nl (8.13.7/8.13.7/Debian-1) with ESMTP id k7UDgKDD023088
	(version=TLSv1/SSLv3 cipher=DHE-RSA-AES256-SHA bits=256 verify=NOT)
	for <frank2@oostum.nl>; Wed, 30 Aug 2006 15:42:20 +0200
Received: (from frank@localhost)
	by suffix.rc.rug.nl (8.13.7/8.13.7/Submit) id k7UDgKka023085
	for frank2@oostum.nl; Wed, 30 Aug 2006 15:42:20 +0200
Resent-From: f.b.brokken@rc.rug.nl
Resent-Date: Wed, 30 Aug 2006 15:42:20 +0200
Resent-Message-ID: <20060830134220.GA23044@rc.rug.nl>
Resent-To: Frank Brokken <frank2@oostum.nl>
Received: from smtp1.rug.nl (smtp1.rug.nl [129.125.50.11])
	by suffix.rc.rug.nl (8.13.7/8.13.7/Debian-1) with SMTP id k7UCs1C1015005
	for <frank@suffix.rc.rug.nl>; Wed, 30 Aug 2006 14:54:01 +0200
Received: from smtp1.rug.nl ([129.125.50.11])
 by smtp1.rug.nl (SMSSMTP 4.1.0.19) with SMTP id M2006083014535607664
 for <frank@suffix.rc.rug.nl>; Wed, 30 Aug 2006 14:53:56 +0200
Received: from mail3.rug.nl (mail3.rug.nl [129.125.50.14])
	by smtp1.rug.nl (8.12.11.20060308/8.12.11) with ESMTP id k7UCrj9O005050
	for <frank@suffix.rc.rug.nl>; Wed, 30 Aug 2006 14:53:45 +0200 (MEST)
Received: from <p108703@rug.nl>
  by mail3.rug.nl (CommuniGate Pro RULES 4.3.9)
  with RULES id 25696695; Wed, 30 Aug 2006 14:53:45 +0200
X-Autogenerated: Mirror
Resent-From: <F.B.Brokken@rug.nl>
Resent-Date: Wed, 30 Aug 2006 14:53:45 +0200
Received: from smtp1.rug.nl ([129.125.50.11] verified)
  by mail3.rug.nl (CommuniGate Pro SMTP 4.3.9)
  with SMTP id 25696640 for f.b.brokken@rug.nl; Wed, 30 Aug 2006 14:53:45 +0200
Received: from smtp1.rug.nl ([129.125.50.11])
 by smtp1.rug.nl (SMSSMTP 4.1.0.19) with SMTP id M2006083014534507641
 for <f.b.brokken@rug.nl>; Wed, 30 Aug 2006 14:53:45 +0200
Received: from spohr.debian.org (spohr.debian.org [140.211.166.43])
	by smtp1.rug.nl (8.12.11.20060308/8.12.11) with ESMTP id k7UCrPPx004944
	for <f.b.brokken@rug.nl>; Wed, 30 Aug 2006 14:53:25 +0200 (MEST)
Received: from debbugs by spohr.debian.org with local (Exim 4.50)
	id 1GIPUV-0003CO-F1; Wed, 30 Aug 2006 05:48:23 -0700
X-Loop: owner@bugs.debian.org
Subject: Bug#385312: bobcat: FTBFS (amd64): cannot convert 'size_t*' to 'unsigned int*' for argument '9' to 'int XQueryPointer(Display*, Window, Window*, Window*, int*, int*, int*, int*, unsigned int*)'
Reply-To: Andreas Jochens <aj@andaco.de>, 385312@bugs.debian.org
Resent-From: Andreas Jochens <aj@andaco.de>
Resent-To: debian-bugs-dist@lists.debian.org
Resent-CC: f.b.brokken@rug.nl (Frank B. Brokken)
Resent-Date: Wed, 30 Aug 2006 12:48:20 +0000
Resent-Message-ID: <handler.385312.B.115694094219647@bugs.debian.org>
X-Debian-PR-Message: report 385312
X-Debian-PR-Package: bobcat
X-Debian-PR-Keywords: patch
Received: via spool by submit@bugs.debian.org id=B.115694094219647
          (code B ref -1); Wed, 30 Aug 2006 12:48:20 +0000
Received: (at submit) by bugs.debian.org; 30 Aug 2006 12:29:02 +0000
Received: from e182064078.adsl.alicedsl.de ([85.182.64.78] helo=kat.ainf.net)
	by spohr.debian.org with esmtp (Exim 4.50)
	id 1GIPBl-00056d-Ge
	for submit@bugs.debian.org; Wed, 30 Aug 2006 05:29:01 -0700
Received: from aj by kat.ainf.net with local (Exim 4.63)
	(envelope-from <aj@andaco.de>)
	id 1GIPBh-0004sf-VQ; Wed, 30 Aug 2006 14:28:58 +0200
To: Debian Bug Tracking System <submit@bugs.debian.org>
From: Andreas Jochens <aj@andaco.de>
Message-Id: <E1GIPBh-0004sf-VQ@kat.ainf.net>
Date: Wed, 30 Aug 2006 14:28:58 +0200
X-Spam-Checker-Version: SpamAssassin 2.60-bugs.debian.org_2005_01_02 
	(1.212-2003-09-23-exp) on spohr.debian.org
X-Spam-Status: NO, hits=-5.00 required=4.00
Resent-Sender: Debian BTS <debbugs@bugs.debian.org>
Resent-Date: Wed, 30 Aug 2006 05:48:23 -0700
X-Spam-Flag: NO
X-Scanned-By: milter-spamc/1.4.366 (smtp1.rug.nl [129.125.50.11]); Wed, 30 Aug 2006 14:53:56 +0200
X-Scanned-By: milter-spamc/1.4.366 (smtp1.rug.nl [129.125.50.11]); Wed, 30 Aug 2006 14:53:45 +0200
X-Spam-Report: Spam detection software on "smtp1.rug.nl".  Questions: postmaster@rug.nl
  Content analysis details:   (-5.0 points, 4.0 required)
  USER_IN_WHITELIST=-5
  ____
X-Spam-Level: 
X-Greylist: Sender IP whitelisted, not delayed by milter-greylist-2.0.2 (oosix.icce.rug.nl [129.125.14.80]); Wed, 30 Aug 2006 15:42:21 +0200 (CEST)
Content-Length: 1065
Lines: 32

Package: bobcat
Version: 1.10.1-1
Severity: serious
Tags: patch

When building 'bobcat' on amd64/unstable,
I get the following error:

g++ -c -o xpointer/o/58get.o -Itmp -Wall -O3 xpointer/get.cc
xpointer/get.cc: In member function 'bool FBB::Xpointer::get(int*, int*) const':
xpointer/get.cc:14: error: cannot convert 'size_t*' to 'unsigned int*' for argument '9' to 'int XQueryPointer(Display*, Window, Window*, Window*, int*, int*, int*, int*, unsigned int*)'
system - failure of system call (status 256)
system - failure of system call (status 256)
make: *** [build-stamp] Error 1

With the attached patch 'bobcat' can be compiled on amd64.

Regards
Andreas Jochens

diff -urN ../tmp-orig/bobcat-1.10.1/xpointer/get.cc ./xpointer/get.cc
--- ../tmp-orig/bobcat-1.10.1/xpointer/get.cc	2006-08-25 14:25:20.000000000 +0000
+++ ./xpointer/get.cc	2006-08-30 11:43:02.000000000 +0000
@@ -5,7 +5,7 @@
     Window rootret;
     Window child;
     Window root = DefaultRootWindow(s_theDisplay);
-    size_t keys;
+    unsigned int keys;
     int dum_x;
     int dum_y;
 

